conf/services/alcohol-duty-returns-frontend.yml

```diff
@@ -0,0 +1,19 @@
+serviceName: the Manage your Alcohol Duty       # The service name that will appear in the title of the accessibility statement.
```

Comment:
Initially I thought the 'the' was a good idea but if you check the screenshot you attached to the PR it now has a double 'the the' in a subsequent section.